src/main/java/nl/rijksoverheid/mev/gezagsmodule/domain/gezagvraag/OudersOverledenOfOnbevoegdTotGezag.java

```diff
@@ -24,13 +36,20 @@ protected OudersOverledenOfOnbevoegdTotGezag(final GezagsBepaling gezagsBepaling
     @Override
     public void perform() {
         preconditieCheckOudersGeregistreerd();
-        boolean ouder1OverledenOfOnbevoegd = gezagsBepaling.getPlOuder1().isOverledenOfOnbevoegd();
-        boolean ouder2OverledenOfOnbevoegd = gezagsBepaling.getPlOuder2().isOverledenOfOnbevoegd();
-        if (ouder1OverledenOfOnbevoegd && ouder2OverledenOfOnbevoegd) {
-            answer = V4A_2_JA_BEIDEN;
-        } else if (ouder1OverledenOfOnbevoegd) {
+
+        var optionalIsOuder1OverledenOfOnbevoegdToken = gezagsBepaling.getPlOuder1().isOverledenOfOnbevoegdEncoded();
```

Comment:
nitpick, je ziet het nu niet omdat je `var` gebruikt maar het type is Optional . nu dat ook in de naam staat is het dubbel er staat nu eigenlijk Optional optional, ik zou optional in de naam weg laten